### PR TITLE
Added uranium glass and uranium window

### PR DESCRIPTION
**Pull request by AsikKEsel**
## About the PR

Aded uranium glass, uranium window, reinforced uranium glass, reinforced uranium window

**Media**

UraniumGlassUpdate

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added uranium glass and uranium window. Don't worry, scientists reduce radioactive emissions by 97.4% with RadAbsorb

### CODE REVIEW

File: Resources/Prototypes/Recipes/Construction/structures.yml [hunk 1]
```diff
@@ -362,6 +362,44 @@
...
 
+- type: construction
+  name: uranium window
+  id: UraniumWindow
+  graph: Window
+  startNode: start
+  targetNode: uraniumWindow
+  category: construction-category-structures
+  canBuildInImpassable: true
+  description: Clear and very tougher, with a RadAbsorb.
```
Comment (deltanedas) [inline Resources/Prototypes/Recipes/Construction/structures.yml:373]: Clear and much tougher than regular glass, with added RadAbsorb (patent pending) to protect you from deadly radiation.

File: Resources/Prototypes/Recipes/Construction/structures.yml [hunk 2]
```diff
@@ -362,6 +362,44 @@
...
+
+- type: construction
+  name: reinforced uranium window
+  id: ReinforcedUraniumWindow
+  graph: Window
+  startNode: start
+  targetNode: reinforcedUraniumWindow
+  category: construction-category-structures
+  canBuildInImpassable: true
+  description: Clear and very tougher, with a RadAbsorb.
```
Comment (deltanedas) [inline Resources/Prototypes/Recipes/Construction/structures.yml:392]: ditto

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by deltanedas** [comment]
has this been added to ore processor recipes, its really handy for making lots of glass sheets

File: Resources/Textures/Structures/Windows/reinforced_uranium_window.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "made by AsikKEsel",
```
Comment (metalgearsloth) [inline Resources/Textures/Structures/Windows/reinforced_uranium_window.rsi/meta.json:4]: Isn't there existing sprites in ss13 to use instead? Also these are derivatives but don't credit the original source.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Arbitrage needs fixing hence the test failure.

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**